javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -490,7 +490,7 @@ public void onComplete(
 
     private static boolean isIgnored(Class<?> c) {
       ClassLoader cl = c.getClassLoader();
-      if (cl instanceof AgentClassLoader || cl instanceof ExtensionClassLoader) {
+      if (cl instanceof AgentClassLoader) {
```

Comment:
What's the reason for this change? Do we want to redefine extension classes?

Reply:
The reason for this change is there is no more `ExtensionClassLoader`. I can change for `MultipleParentClassLoader`, but there may be different instance of it, not created by us.

And I don't know if there is any actual implication of this check.

Reply:
But do we actually need this?

Reply:
It's an optimization to avoid retransforming classes that were loaded by the extension before we set up the bytebuddy transformers, @laurit's idea seems simple enough to be worth keeping.

Reply:
Done.